.circleci/config.yml

```diff
@@ -2199,11 +2199,11 @@ jobs:
           export PATH=$JAVA_HOME/bin:$PATH
           cd ~/cassandra
           mkdir ~/dtest_jars
-          git remote add apache git://github.com/apache/cassandra.git
-          for branch in cassandra-2.2 cassandra-3.0 cassandra-3.11 trunk; do
+          git remote add maedhroz git@github.com:maedhroz/cassandra.git
+          for branch in cassandra-2.2 CASSANDRA-16387-3.0 CASSANDRA-16387-3.11 trunk; do
             # check out the correct cassandra version:
-            git remote set-branches --add apache '$branch'
-            git fetch --depth 1 apache $branch
+            git remote set-branches --add maedhroz '$branch'
+            git fetch --depth 1 maedhroz $branch
```

Comment:
@adelapena @ekaterinadimitrova2 This commit will just be reverted once CASSANDRA-16387 commits...